### PR TITLE
Api v3 members port

### PR DESCRIPTION
**Pull request by TheHollidayInn**
Fixes put_issue_url_here
### Changes
- *_Note: *_ Merge this after the api-v3-port-groups
- Update Member service to use API v3
- Update some controller dependent on the Members service

---

UUID:

### CODE REVIEW

File: test/spec/services/memberServicesSpec.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-describe('memberServices', function() {
+describe.only('memberServices', function() {
```
Comment (paglias) [inline test/spec/services/memberServicesSpec.js]: this should be removed

File: website/public/js/app.js [hunk 2]
```diff
@@ -150,12 +150,16 @@ window.habitrpg = angular.module('habitrpg',
...
+          controller: ['$scope', 'Groups', 'Chat', '$stateParams', 'Members',
+          function($scope, Groups, Chat, $stateParams, Members){
             Groups.Group.get($stateParams.gid)
               .then(function (response) {
                 $scope.group = response.data.data;
                 Chat.seenMessage($scope.group._id);
+                Members.getGroupMembers($scope.group._id)
+                  .then(function (response) {
+                    $scope.group.members = response.data.data;
+                  });
```
Comment (paglias) [inline website/public/js/app.js:162]: it should also fetch invites?

File: test/spec/controllers/partyCtrlSpec.js [hunk 3]
```diff
@@ -136,6 +143,25 @@ describe("Party Controller", function() {
     });
   });
 
+  describe.only("create", function() {
```
Comment (crookedneighbor) [inline test/spec/controllers/partyCtrlSpec.js]: Only.

**Comment by crookedneighbor** [comment]
Looks reasonable.

**Comment by TheHollidayInn** [comment]
Updated!

File: website/public/js/app.js [hunk 4]
```diff
@@ -150,12 +150,17 @@ window.habitrpg = angular.module('habitrpg',
...
-              $scope.group = group;
-              Chat.seenMessage(group._id);
-            });
+          controller: ['$scope', 'Groups', 'Chat', '$stateParams', 'Members',
+          function($scope, Groups, Chat, $stateParams, Members){
+            Groups.Group.get($stateParams.gid)
+              .then(function (response) {
+                $scope.group = response.data.data;
+                Chat.seenMessage($scope.group._id);
+                Members.getGroupMembers($scope.group._id)
```
Comment (paglias) [inline website/public/js/app.js:159]: what about invites? We can decide not to fetch them unless explicitly requested but we should decide that

Comment (TheHollidayInn) [inline website/public/js/app.js:159]: Where do we display invites on the UI? In the member section? If so, we should definitely fetch them here.

Comment (paglias) [inline website/public/js/app.js:159]: In the members box, if you scroll down there's a list of invited user: 
schermata 2016-04-25 alle 21 34 17

Comment (TheHollidayInn) [inline website/public/js/app.js:159]: Awesome. I'll update this soon, then.

**Comment by TheHollidayInn** [comment]
Updated.. The failing tests are server side. I'm unsure if my code would affect them.

**Comment by paglias** [comment]
Rerunning the tests, sometimes lately they've been failing for no reason

File: website/public/js/controllers/guildsCtrl.js [hunk 5]
```diff
@@ -69,6 +68,12 @@ habitrpg.controller("GuildsCtrl", ['$scope', 'Groups', 'User', 'Challenges', '$r
 
       $scope.reject = function(guild) {
         Groups.Group.rejectInvite(guild.id);
+
+        var i = _.findIndex(User.user.invitations.guilds, {id: guild.id});
+        if (i !== -1) {
+          User.user.invitations.guilds.splice(i, 1);
+          User.set({'invitations.guilds':User.user.invitations.guilds});
```
Comment (paglias) [inline website/public/js/controllers/guildsCtrl.js]: my fault here... sorry. rejectInvite will handle everything so you can remove these lines!

**Comment by TheHollidayInn** [comment]
Looks like this is passing now.

**Pull request merged**